apps/client/src/components/Layouts/DefaultLayout.tsx

```diff
@@ -1,28 +1,14 @@
-import React, { useEffect } from "react";
-import { useHistory } from "react-router-dom";
+import React from "react";
 
 import withTracking from "../../hoc/withTracking";
 import BaseLayout from "./BaseLayout";
 
 interface DefaultLayoutProps {
   heading: string;
   children: React.ReactNode;
-  matomoPageView: Function;
 }
 
-function DefaultLayout({
-  heading,
-  children,
-  matomoPageView,
-}: DefaultLayoutProps) {
-  const { location } = useHistory();
-
-  useEffect(() => {
-    matomoPageView();
-    // @TODO: We need to fix this!
-    //eslint-disable-next-line
-  }, [location.pathname]);
-
+function DefaultLayout({ heading, children }: DefaultLayoutProps) {
```

Comment:
If this page does nothing other than passing it's own props, then we can remove it right?